flink-core/pom.xml

```diff
@@ -165,6 +165,7 @@ under the License.
 							<exclude>org.apache.flink.configuration.ConfigConstants#ENABLE_QUARANTINE_MONITOR</exclude>
 							<exclude>org.apache.flink.configuration.ConfigConstants#NETWORK_REQUEST_BACKOFF_INITIAL_KEY</exclude>
 							<exclude>org.apache.flink.configuration.ConfigConstants#NETWORK_REQUEST_BACKOFF_MAX_KEY</exclude>
+							<exclude>org.apache.flink.configuration.ConfigConstants#DEFAULT_TASK_MANAGER_IPC_PORT</exclude>
```

Comment:
Adding an exclusion here should only be the last resort; converting the old constant to the new one is trivial.